package.json

```diff
@@ -28,6 +27,7 @@
   ],
   "devDependencies": {
     "chai": "3.5.0",
+    "flow-bin": "0.33.0",
```

Comment:
@kumar303 yep, the only reason to have flow-bin in the dev dependency is that we need it to run the tests.

I can change it to `^0.33.0`, but I though that it would be better to keep the exact version so that greenkeeper can help us to keep it tested on new flow-bin releases by automatically creating the its "npm updates pull requests".